tools/ci_build/github/azure-pipelines/android-arm64-v8a-QNN-crosscompile-ci-pipeline.yml

```diff
@@ -1,10 +1,24 @@
+parameters:
+
+- name: QnnSdk
+  displayName: QNN SDK version
+  type: string
+  default: qnn-v2.8.0.230223123141_52150
+  values:
+  - qnn-v2.6.0.221227110525_42395
+  - qnn-v2.8.0.230223123141_52150
+
 jobs:
 - job: Build_QNN_EP
-  pool: Onnxruntime-QNNEP-Ubuntu-2004-CPU
```

Comment:
the pool is changing?

Reply:
Yes, the original pool is not accessible in the onnxruntime Azure DevOps Organization. All the original pools in the internal repo are created in the aiinfra org.

Reply:
A few more details for clarity:
The Linux and Android pipelines use the same pool. The Onnxruntime-QNNEP-Ubuntu-2004-CPU pool _could_ have been moved to the new org. However, since the linux pipeline was setup while we were transitioning to github, I didn't want to prevent us from being able to run the linux pipeline from the internal repo.

I won't have to create new pools for the other pipelines. I can move them.